src/main/resources/db/migration/changes/ZZ-LinkingTableChanges.json

```diff
@@ -0,0 +1,47 @@
+{
+    "databaseChangeLog": [
+        {
+            "changeSet": {
+                "id": "ZZ-1",
```

Comment:
Could we pick a slightly more meaningful id? It's not clear to students how they should copy and adapt this if they copy this changelog.